src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -86,6 +92,10 @@ var ReactCSSTransitionGroupChild = React.createClass({
       }
     };
 
+    if (childRef.componentWillTransition) {
```

Comment:
I needed to change this to `if (childRef && childRef.componentWillTransition) {}` to prevent errors in certain situations.